code/game/objects/items/storage/belt.dm

```diff
@@ -753,7 +773,7 @@
 	for(var/i = 1 to storage_slots)
 		new /obj/item/weapon/throwing_knife(src)
 
-/obj/item/storage/belt/knifepouch/_item_insertion(obj/item/W, prevent_warning = 0)
+/obj/item/storage/belt/knifepouch/_item_insertion(obj/item/W, prevent_warning = FALSE)
```

Comment:
```suggestion
/obj/item/storage/belt/knifepouch/_item_insertion(obj/item/new_item, prevent_warning = FALSE)
```

The other `obj/item/W` you modified because of changing `0` to `FALSE` are okay to leave until those procs actually get modified.